source/test/fixtures/failure-strict-null-checks-as-default-config-value/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think we should move this to a `override-config` fixture or something so it's clear that the this fixture does that.